src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor.cs

```diff
@@ -59,26 +58,19 @@ protected override async Task OnInitAsync()
 
     private async Task LoadEditProfileData()
     {
-        User = await StateService.GetValue($"{nameof(EditProfile)}-{nameof(User)}", async () =>
-            await HttpClient.GetFromJsonAsync("User/GetCurrentUser", TodoTemplateJsonContext.Default.UserDto));
+        User = (await StateService.GetValue($"{nameof(EditProfile)}-{nameof(User)}", async () =>
+            await HttpClient.GetFromJsonAsync("User/GetCurrentUser", TodoTemplateJsonContext.Default.UserDto))) ?? new();
 
-        UserToEdit.FullName = User?.FullName;
-        UserToEdit.BirthDate = User?.BirthDate;
-        UserToEdit.Gender = User?.Gender;
+        UserToEdit.FullName = User.FullName;
+        UserToEdit.BirthDate = User.BirthDate;
+        UserToEdit.Gender = User.Gender;
     }
 
-    private void CheckSaveButtonEnabled()
-    {
-        if (User?.FullName == UserToEdit.FullName &&
-            User?.BirthDate == UserToEdit.BirthDate &&
-            User?.Gender == UserToEdit.Gender)
-        {
-            IsSaveButtonEnabled = false;
-            return;
-        }
-
-        IsSaveButtonEnabled = true;
-    }
+    private bool IsSubmitButtonEnabled =>
+            ((User.FullName ?? String.Empty) != (UserToEdit.FullName ?? String.Empty)
```

Comment:
Is it correct to use the String class? Or use alias it
@msynk

Reply:
it's better to use the alias `string` and not the class itself. actually, it should be changed to use the alias.
never use the actual classes when there is an alias in C# for them.